src/content/docs/apis/nerdgraph/examples/nerdgraph-historical-data-export.mdx

```diff
@@ -17,24 +17,27 @@ When running a [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/
 
 ## Limits and restrictions [#limits-restrictions]
 
-Here are some limits and restrictions on the feature: 
+Here are some limits and restrictions on the feature:
 
 ### Usage limits [#usage-limits]
 
-The following are the default usage limits for an export: 
+The following are the default usage limits for an export:
 * Exports should be estimated to return fewer than 200 million events
 * Exports should be estimated to inspect fewer than 5 billion events
 * No more than two concurrent exports per account
 
-If you'd like higher limits, talk to your account representative. 
+If you'd like higher limits, talk to your account representative.
 
 ### Data type restrictions [#data-type-restrictions]
 
-[Metric timeslice data](/docs/data-apis/understand-data/new-relic-data-types/#timeslice-data) is not available for export. 
+* [Metric timeslice data](/docs/data-apis/understand-data/new-relic-data-types/#timeslice-data) is not available for export.
+* [FedRAMP data](/docs/security/security-privacy/compliance/certificates-standards-regulations/fedramp-moderate/)
+* [Blob attributes](/docs/logs/ui-data/long-logs-blobs/)
+* Array attributes
```

Comment:
These are the main change

Reply:
@rbayersdorfer-newrelic could you clarify what we mean by "restrictions" here? Are we saying that only these data types are allowed, or are we saying that these are all prohibited?

What's the take-away for "Array attributes"? What should users know?

Reply:
@rhetoric101 I've updated to add line 33, does that answer your question?

Reply:
Thanks @rbayersdorfer-newrelic !